### PR TITLE
Added conda recipe for travis-encrypt

### PR DESCRIPTION
**Pull request by mandeep**
This is a recipe for travis-encrypt. travis-encrypt is an application that encrypts passwords and environmen variables in .travis.yml for use with Travis CI.

### CODE REVIEW

File: recipes/travis-encrypt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package:
+  name: travis-encrypt
+  version: "0.6.0"
+
+source:
+  git_url: https://github.com/mandeep/Travis-Encrypt.git
```
Comment (synapticarbors) [inline recipes/travis-encrypt/meta.yaml]: Please build from pypi or the github release. See:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

File: recipes/travis-encrypt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  noarch: python
```
Comment (synapticarbors) [inline recipes/travis-encrypt/meta.yaml]: I don't think conda-forge currently supports `noarch`.

File: recipes/travis-encrypt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  entry_points:
+    - travis-encrypt=travis.cli:cli
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/travis-encrypt/meta.yaml]: It looks like only `python` and `setuptools` are required to build the recipe.

File: recipes/travis-encrypt/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+  commands:
+    - travis-encrypt --help
+
+about:
+  home: https://www.github.com/mandeep/Travis-Encrypt
+  license: GNU General Public License v3 or later (GPLv3+)
+  summary: 'A command line application that encrypts passwords for use with Travis CI.'
+  license_family: GPL3
+extra:
```
Comment (synapticarbors) [inline recipes/travis-encrypt/meta.yaml]: Add a blank like above  the `extra` section.

File: recipes/travis-encrypt/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+package:
+  name: travis-encrypt
+  version: "0.6.0"
```
Comment (synapticarbors) [inline recipes/travis-encrypt/meta.yaml]: Unless there is a compelling reason, update to the newest release (which looks like `0.7.1`).

File: recipes/travis-encrypt/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  imports:
+    - travis
+    - travis.tests
+
+  commands:
+    - travis-encrypt --help
+
+about:
+  home: https://www.github.com/mandeep/Travis-Encrypt
+  license: GNU General Public License v3 or later (GPLv3+)
```
Comment (synapticarbors) [inline recipes/travis-encrypt/meta.yaml]: `license: GPL-3.0`

Also for GPL, we typically require a `license_file` to be specified. See:

https://github.com/mandeep/Travis-Encrypt/issues/4

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

File: recipes/travis-encrypt/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - setuptools
+    - click >=6.6
+    - cryptography >=1.7.1
+    - pyyaml >=3.12
+    - requests >=2.12.5
+
+  run:
+    - python
+    - setuptools
```
Comment (synapticarbors) [inline recipes/travis-encrypt/meta.yaml]: Is `setuptools` required to run the library? If not, please remove it.

File: recipes/travis-encrypt/meta.yaml (hunk 5 above)
Comment (mandeep) [inline recipes/travis-encrypt/meta.yaml]: The current release at the time was 0.6.0. I'll change it to 0.7.2.

**Comment by mandeep** [comment]
@synapticarbors thanks for the review! It seems that all checks have passed. Let me know if you would like me to squash the commits.

**Comment by synapticarbors** [comment]
@mandeep The recipe still looks to be missing the `license_file` entry.

**Comment by mandeep** [comment]
Oops sorry about that. I've added it in the latest commit.

**Comment by synapticarbors** [comment]
@mandeep -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**